pkg/innerring/config/contracts.go

```diff
@@ -0,0 +1,23 @@
+package config
+
+type Contracts struct {
+	NeoFS      string   `mapstructure:"neofs"`
+	Processing string   `mapstructure:"processing"`
+	Audit      string   `mapstructure:"audit"`
+	Balance    string   `mapstructure:"balance"`
+	Container  string   `mapstructure:"container"`
+	Netmap     string   `mapstructure:"netmap"`
+	Proxy      string   `mapstructure:"proxy"`
+	Reputation string   `mapstructure:"reputation"`
+	Alphabet   Alphabet `mapstructure:"alphabet"`
+}
+
+type Alphabet struct {
+	Az      string `mapstructure:"az"`
```

Comment:
Wow. How many times have we used these settings? Theoretically, this list should support all of `pkg/util/glagolitsa`, but my proposal is to drop it completely (@cthulhu-rider, @carpawell).